### PR TITLE
Allow to override session timeout.

### PR DESCRIPTION
**Pull request by marcosnils**
This comes useful as it can be overriden from the SDK now

@xetorthio :bell:

### CODE REVIEW

File: handlers/new_session.go [hunk 1]
```diff
@@ -9,14 +9,19 @@ import (
 )
 
 func NewSession(rw http.ResponseWriter, req *http.Request) {
-	if !services.IsHuman(req) {
+	req.ParseForm()
+	challenge := req.Form.Get("g-recaptcha-response")
+	if !services.IsHuman(challenge) {
```
Comment (xetorthio) [inline handlers/new_session.go]: Why did you make this change? `IsHuman` tells nothing about recaptcha, so it makes a lot more sense to send the whole request.

File: services/session.go [hunk 2]
```diff
@@ -202,24 +201,20 @@ func CloseSession(s *Session) error {
...
-func getExpiryHours() int {
-	hours := 4
-	override := os.Getenv("EXPIRY")
-	if len(override) > 0 {
-		value, err := strconv.Atoi(override)
-		if err == nil {
-			hours = value
+var defaultDuration = 4 * time.Hour
+
+func GetDuration(reqDur string) time.Duration {
```
Comment (xetorthio) [inline services/session.go]: Why did you remove the environment variable override?

File: handlers/new_session.go (hunk 1 above)
Comment (marcosnils) [inline handlers/new_session.go]: You're right,  I took the logic outside the function mainly because parse form needs to be done outside. I can still send the request anyways and parse the form in the NewSession function.

Does that sound right?

**Comment by xetorthio** [comment]
Yeap. Much better

**Comment by marcosnils** [comment]
@xetorthio plz review.
